### PR TITLE
Install missing dependency

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

**Comment by EPashkin** [comment]
I already tried that, it already present in system,
see: https://github.com/gtk-rs/gtk/pull/795#issuecomment-470670457

File: .travis.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ addons:
     - libmount-dev
 before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install gtk+3 cairo atk; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install libffi gtk+3 cairo atk; fi
```
Comment (sdroege) [inline .travis.yml]: libffi is a dependency of glib, so this is only a workaround for brew being broken. Can you add a comment about that here?

Comment (GuillaumeGomez) [reply .travis.yml]: Sure!

**Comment by GuillaumeGomez** [comment]
@EPashkin Damn, I completely forgot about it... Do you know if there is an open issue somewhere about this?

**Comment by EPashkin** [comment]
IMHO it apveyor image problem, it need be refreshed.
Fast search did not show same problem 2 days ago or any way disable brew's autocleanup

**Pull request closed without merging**